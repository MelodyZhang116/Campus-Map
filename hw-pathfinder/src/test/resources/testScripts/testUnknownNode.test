# test path find with unknown start or destination

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddEdge g n1 n2 135.798463321

FindPath g n1 n2

FindPath g n1 n3

FindPath g n1 x

FindPath g x n1

FindPath g x y

FindPath g y x

FindPath g x x

FindPath g n1 n1