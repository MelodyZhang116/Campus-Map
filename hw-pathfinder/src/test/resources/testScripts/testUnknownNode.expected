# test path find with unknown start or destination

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added edge 135.798 from n1 to n2 in g

path from n1 to n2:
n1 to n2 with weight 135.798
total cost: 135.798

path from n1 to n3:
no path found

unknown: x

unknown: x

unknown: x
unknown: y

unknown: y
unknown: x

unknown: x
unknown: x

